ProofPoint/proofpoint-pod/ingest/parser.yml

```
@@ -95,6 +95,7 @@ stages:
       - set:
           event.category: ["email"]
           event.dataset: "msgParts"
+          event.action: "{{json_event.message.msgParts.disposition}}"
```

The disposition field is at the root of the event
```suggestion
          event.action: "{{json_event.message.disposition}}"
```